autometrics/src/prometheus_exporter.rs

```diff
@@ -107,11 +136,11 @@ impl GlobalPrometheus {
 /// ```
 /// # fn main() {
 /// # #[cfg(feature="prometheus-exporter")]
-///     let _exporter = autometrics::global_metrics_exporter();
+///     autometrics::prometheus_exporter::init();
 /// # }
 /// ```
-pub fn global_metrics_exporter() -> GlobalPrometheus {
-    GLOBAL_EXPORTER.clone()
+pub fn init() {
+    let _exporter = GLOBAL_EXPORTER.clone();
```

Comment:
wont this be dropped immediately?

Reply:
Yes, but it'll cause the `once_cell::Lazy` to be initialized